### PR TITLE
Adding Send dialog

### LINKED ISSUE #4211: Learner Record: Send to partner institutions
As a learner, I would like to send my learner record to the MicroMasters Pathway partner school that I am applying to. 

As a Pathway partner school, I expect to receive a system-generated email that includes the learner record information. I would prefer to receive an email directly from MIT instead of information that the learner is passing along. 

#### Designs and Mock-ups

Web UI based on edX's current design. I sent screenshots to Anna. 
[Email template](https://goo.gl/maps/RxNQjAbLHCQ2) is below in comments

#### Requirements

- [ ] partner data based on info in [ZenDesk #26552](https://odl.zendesk.com/agent/tickets/26552)

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
Fix #4211

#### What's this PR do?
Adding option to send email to partner school with link to transcript.

#### How should this be manually tested?
Go to your program records page and try to send the records to one of the predefined pathway partner schools.

Screen Shot 2019-03-20 at 3 13 55 PM
Screen Shot 2019-03-20 at 3 14 06 PM
Screen Shot 2019-03-20 at 3 14 19 PM

### CODE REVIEW

**Comment by pdpinch** [comment]
This isn't passing tests -- is it because of the debian issue with docker, that #4271 is supposed to fix?

**Comment by pdpinch** [comment]
@tasawernawaz can you review this?

File: mail/models.py [hunk 1]
```diff
@@ -64,3 +64,12 @@ def __str__(self):
...
         )
+
+
+class PartnerSchool(models.Model):
+    """
+    Models for partner school to send records to
+    """
+
+    name = models.CharField(max_length=255)
+    email = models.TextField(null=False)
```
[the remark this answers is not in the document]
Comment (annagav) [reply mail/models.py]: The other models use TextField for email so I am keeping it consistent.

**Comment by annagav** [comment]
@tasawernawaz I addressed you comments. Thanks!

**Comment by tasawernawaz** [comment]
@annagav changes looks good. However I cant run code on my local machine, because I have broken up my local container, as I am working on updating docker to stretch.

CC: @pdpinch I am not sure, if we can merge this before finishing #4294

**Comment by pdpinch** [comment]
@tasawernawaz I think you can resume your review of this?

**Comment by tasawernawaz** [comment]
Sure, I will.

**Comment by tasawernawaz** [comment]
@annagav Changes looks good, please rebase it against master to make sure that it is working with docker update.

**Pull request merged**